.github/workflows/atlas.yml

```diff
@@ -0,0 +1,58 @@
+name: Atlas
```

Comment:
How does this work? You built a custom action for this? Pretty neat! But that means you'll have to be an owner of each of these modules, correct?

Reply:
Correct, we can pick someone's key to create all the modules on atlas. I did mine because it was faster. And yes I will need to be part of the owners of the modules. 

It would be nice to use the bots key but there isnt support for this

Reply:
Gotcha, pretty cool. Thanks for doing this.